src/FileSystemUtilities.js

```diff
@@ -65,6 +65,8 @@ export default class FileSystemUtilities {
 
   @logger.logifySync
   static unlinkSync(filePath) {
-    fs.unlink(filePath);
+    if (pathExists(filePath)) {
```

Comment:
Good question.  Presumably if we're calling `unlinkSync` we should be reasonably sure that there's something to unlink, and it's an exception otherwise.  I think that's the logic behind the behavior of the `*Sync` methods in node's standard library.